ush/mesoscale/evs_prepare_sref.sh

```diff
@@ -213,9 +245,10 @@ export output_base=${WORK}/pb2nc
      export vbeg=${vhr}
      export vend=${vhr}
 
-     #Split the prepbufr data files into specifiically required data types to reduce
-     #the walltime
-     #use bufr module tool: split_by_subset
+     #*******************************************************************************
+     # Using the bufr module tool: split_by_subset to split the prepbufr data files
+     #  into specifiically required data types to reduce the walltime
```

Comment:
Hi Binbin, another small typo error. Please remove the 3rd "i" in "specifically" on line 250.